### PR TITLE
[GITHUB-20] Remove configure tag

### PR DESCRIPTION
**Pull request by lobsterdore**
Installing users in both build and configure adds unnessaccary
steps to machine boot up, if the user is already installed
during the build phase then there is no need to re-run the
installation during configure.

### CODE REVIEW

**Comment by wojtek-oledzki** [comment]
I Like

I Like

File: README.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ and run `ansible-galaxy install -p ./roles -r roles.yml`
 
 ## Tags
 
-This role uses three tags: **build**, **configure** and **maintain**
+This role uses three tags: **build** and **maintain**
```
Comment (chroche) [inline README.md]: It's two now :-)

File: README.md [hunk 2]
```diff
@@ -45,9 +45,9 @@ and run `ansible-galaxy install -p ./roles -r roles.yml`
...
 
-* `build` & `configure` - Ensures that specified groups and users are
+* `build` - Ensures that specified groups and users are
   present.
 * maintain - Ensures users on an already built and configured instance
```
Comment (chroche) [inline README.md]: maintain should be formatted like build

**Review by chroche — COMMENTED** [review]
Minor fixes to README file

**Pull request merged**